bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -441,7 +441,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <reset_snow_glc_ela>1.e9</reset_snow_glc_ela>
 
 <snow_cover_fraction_method>SwensonLawrence2012</snow_cover_fraction_method>
+
```

Comment:
should we be specific about different physics versions (4.5, 5.0, 5.1) and avoid setting a default?  My only thought here is to make sure we identify the default when we start working on the > 5.1 series? Happy to defer to convention used elsewhere in the namelist files

Reply:
I'm not sure what line(s) you are pointing to.  Are you talking about these lines?

`<snow_thermal_cond_method>Jordan1991</snow_thermal_cond_method>`
`<snow_thermal_cond_method phys="clm5_1">Sturm1997</snow_thermal_cond_method>`

I was following the recent example for Meier:

`<z0param_method>ZengWang2007</z0param_method>`
`<z0param_method phys="clm5_1">Meier2022</z0param_method>`

Reply:
I guess your point is, should we be doing something like this:

`<snow_thermal_cond_method phys="clm4_5>Jordan1991</snow_thermal_cond_method>`
`<snow_thermal_cond_method phys="clm5_0>Jordan1991</snow_thermal_cond_method>`
`<snow_thermal_cond_method phys="clm5_1">Sturm1997</snow_thermal_cond_method>`

Reply:
I agree with @olyson and @wwieder I think the default should be explicit for each clm physics version.